### DISCUSSION TITLE
Show placeholder with dropdown input plugin && Hide overflow of selected option under clear button

### DISCUSSION THREAD

**Discussion by martinbarilik** [orchidjs/tom-select#287] [Ideas]
Hi, i have two proposals.

1. I need to show placeholder with dropdown input  plugin. The select is empty when selected option is cleared out. 
2. Can you make long texts in selected options to hide overflow under clear box ? Use example with long text to see the problem.

example: https://jsfiddle.net/mArtinko5MB/gj7fdhxa/8/

**Comment by oyejorge** [comment]
[Here's an example with a little CSS ](https://jsfiddle.net/yo6Lagz4/1/)

**Comment by martinbarilik** [comment]
Even tho i appreciate the effort, it's not an option for me. I have pages with more then 20 selects, every of them if different and has different placeholder. I've got like 100 of those pages, so to make placeholder being set in scss is not quite what i am searching for :) I need it to be  set dynamically ( by data attr f.e. )

**Comment by oyejorge** [comment]
[Another option](https://jsfiddle.net/m9jywrn4/)

**Comment by martinbarilik** [comment]
This works great. The library is awesome ;) customization with infinite options ...

Thanks
